### PR TITLE
Added title feature for each coordinate on a scatter plot.

### PR DESCRIPTION
**Pull request by RafaelMiquelino**
Included the possibility to give names on each mark on a scatter plot. Interesting when loading a set of coordinates with the respective names.

### CODE REVIEW

**Comment by otreidel** [comment]
This would be a great feature. However, you've change the API for `scatter` in a way that makes it backwards incompatible. Moreover, I think that `titles` should be an optional argument not a mandatory one.

**Comment by aa-rons** [comment]
Has this feature been implemented?

**Comment by vgm64** [comment]
[on train] Since it was an api breaking change, I doubt it. New/updated PRs
welcome. Definitely a requested feature.

File: gmplot/gmplot.py [hunk 1]
```diff
@@ -45,22 +45,22 @@ def geocode(self, location_string):
     def grid(self, slat, elat, latin, slng, elng, lngin):
         self.gridsetting = [slat, elat, latin, slng, elng, lngin]
 
-    def marker(self, lat, lng, color='#FF0000', c=None, title="no implementation"):
+    def marker(self, lat, lng, title="no implementation", color='#FF0000', c=None):
```
Comment (frslm) [inline gmplot/gmplot.py:48]: Let's revert this to keep the order of arguments as is.

File: gmplot/gmplot.py [hunk 2]
```diff
@@ -45,22 +45,22 @@ def geocode(self, location_string):
...
         if c:
             color = c
         color = self.color_dict.get(color, color)
         color = self.html_color_codes.get(color, color)
         self.points.append((lat, lng, color[1:], title))
 
-    def scatter(self, lats, lngs, color=None, size=None, marker=True, c=None, s=None, **kwargs):
+    def scatter(self, lats, lngs, titles, color=None, size=None, marker=True, c=None, s=None, **kwargs):
```
Comment (frslm) [inline gmplot/gmplot.py:55]: Let's move `titles` to the end (right before `kwargs`) as an optional parameter that defaults to `None`, to preserve the existing parameter order.

**Comment by frslm** [comment]
Will close this since it's been implemented as part of #152, which also added several other properties to `scatter()`.

**Pull request closed without merging**